snappy_pipeline/workflows/tumor_mutational_burden/__init__.py

```diff
@@ -25,8 +28,11 @@
 step_config:
     tumor_mutational_burden:
```

Comment:
The choice between somatic variant calling & annotation is input several times. This is not good.
Ideally, the config file should just ask for `path_somatic_variant` and `has_annotation` with `True/False` option.
Based on the `has_annotation` input, `__init__.py` would then select the workflow, and the annotation tools.